### PR TITLE
feat: [SRM-14791]: Roundoff slider time to nearest 5 mins

### PR DESCRIPTION
**Pull request by deepesh-ui**
### Summary

Move the slider in SLO graph and service health to 5 minute boundary

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/__tests__/ServiceHealth.test.tsx [hunk 1]
```diff
@@ -213,7 +213,7 @@ describe('Unit tests for ServiceHealth', () => {
     const endXPercentage = 0.35275080906148865
 
     expect(calculateStartAndEndTimes(startXPercentage, endXPercentage, mockedTimestamps)).toEqual([
-      1630863277338, 1630887233649
+      1630863300338, 1630887300649
```
Comment (karansaraswat19) [inline src/modules/85-cv/pages/monitored-service/components/ServiceHealth/__tests__/ServiceHealth.test.tsx:216]: Shouldn't we round off milliseconds too? eg: 1630863300000

Comment (deepesh-ui) [reply src/modules/85-cv/pages/monitored-service/components/ServiceHealth/__tests__/ServiceHealth.test.tsx:216]: will not make much difference so didn't round off milliseconds

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/ServiceHealth.utils.ts [hunk 2]
```diff
@@ -125,15 +125,22 @@ export const getTimeFormatMoment = (format?: string): string => {
   return timeFormat
 }
 
+export const nearest5Minutes = (someMoment: moment.Moment) => {
+  const roundedMinutes = Math.ceil(someMoment.clone().minute() / 5) * 5
```
Comment (harshilbits) [inline src/modules/85-cv/pages/monitored-service/components/ServiceHealth/ServiceHealth.utils.ts:129]: Please cover this in testcase.

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/ServiceHealth.utils.ts [hunk 3]
```diff
@@ -125,15 +125,22 @@ export const getTimeFormatMoment = (format?: string): string => {
...
   endXPercentage: number,
   timestamps?: number[]
 ): [number, number] | undefined {
   if (!timestamps?.length) return
   const startTime = Math.floor(startXPercentage * (timestamps[timestamps.length - 1] - timestamps[0]) + timestamps[0])
+  const nearest5MinutesStartTime = nearest5Minutes(moment(startTime)).valueOf()
   const endTime = Math.floor(endXPercentage * (timestamps[timestamps.length - 1] - timestamps[0]) + timestamps[0])
-  return [startTime, endTime]
+  const nearest5MinutesEndTime = nearest5Minutes(moment(endTime)).valueOf()
+  return [nearest5MinutesStartTime, nearest5MinutesEndTime]
```
Comment (harshilbits) [inline src/modules/85-cv/pages/monitored-service/components/ServiceHealth/ServiceHealth.utils.ts:143]: same.

Comment (deepesh-ui) [reply src/modules/85-cv/pages/monitored-service/components/ServiceHealth/ServiceHealth.utils.ts:143]: updated existing testcase

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/ServiceHealth.utils.ts (hunk 2 above)
Comment (deepesh-ui) [reply src/modules/85-cv/pages/monitored-service/components/ServiceHealth/ServiceHealth.utils.ts:129]: removed function  nearest5Minutes

**Review by harshilbits — APPROVED** [review]

**Pull request merged**